lua/rose-pine.lua

```diff
@@ -126,7 +126,7 @@ local function set_highlights()
 		RedrawDebugClear = { fg = palette.base, bg = palette.gold },
 		RedrawDebugComposed = { fg = palette.base, bg = palette.pine },
 		RedrawDebugRecompose = { fg = palette.base, bg = palette.love },
-		Search = { fg = palette.base, bg = palette.text },
+		Search = { fg = palette.base, bg = palette.rose },
```

Comment:
Let's leave this how it was, having two colours for search and current search is a little overwhelming in my opinion

Reply:
May I suggest a modification to this for the theme overall, as I feel the default background for `Search` is terrible. The white `text` seems to glaring to me, the gold for `CurSearch` is ok, but could we try toning down the overall results background?

Reply:
Yea that's completely fine. Let's discuss in a new issue https://github.com/rose-pine/neovim/issues/305